src/home/homework-luks.c

```diff
@@ -1229,7 +1229,7 @@ int home_setup_luks(
                 PasswordCache *cache,
                 UserRecord **ret_luks_home) {
 
-        sd_id128_t found_partition_uuid = SD_ID128_NULL, found_luks_uuid = SD_ID128_NULL, found_fs_uuid = SD_ID128_NULL;
+        sd_id128_t found_partition_uuid, found_luks_uuid, found_fs_uuid;
```

Comment:
found_luks_uuid may not be initialized, as `luks_open()` is called only when `!setup->crypt_device`.

Reply:
Arggh, indeed. Should be better now.